README.md

```diff
@@ -108,7 +115,158 @@ Log in with psql shell to cnx-db
+
+* Configure rclone with and create a new remote (n):
+
+```
+rclone config
+```
+
+![rcloneconfig](https://user-images.githubusercontent.com/1050582/62883463-c27a6780-bd3c-11e9-83f3-21bc0a037cbd.jpg)
+
+* Name it e.g. `openstax` and choose Amazon S3:
+
+![rclones3](https://user-images.githubusercontent.com/1050582/62883464-c312fe00-bd3c-11e9-9d90-404bde984996.jpg)
+
+* Choose S3
+* Enter AWS credentials in next step
+* Now enter your access key id and secret access key inside:
+
+![rclonesecret](https://user-images.githubusercontent.com/1050582/62883465-c312fe00-bd3c-11e9-94aa-eaba436bd5b7.jpg)
+
+* Choose Ohio: `us-east-2`
+* Use default entpoint
+* Choose again Ohio: `us-east-2`
+* Choose FULL_CONTROL for owner (default):
+
+![rclonefullcontrol](https://user-images.githubusercontent.com/1050582/62883466-c312fe00-bd3c-11e9-8c12-b7ee4c68c7fd.jpg)
+
+* Choose no server side encryption
+* No KMS ID:
+
+![rclonekms](https://user-images.githubusercontent.com/1050582/62883467-c312fe00-bd3c-11e9-97f7-6c0947e16aa6.jpg)
+
+* Default storage class
+* No advanced config
+
+* At the end confirm all your settings which should look similar to this:
+
+![rcloneconfirm](https://user-images.githubusercontent.com/1050582/62883469-c3ab9480-bd3c-11e9-819c-59c2d222adea.jpg)
+
+* quit config
+
+#### Test that rclone works
+
+Run following command to remote `openstax` look at the buckets available:
+
+```
+rclone lsd openstax:
+```
+
+You should see a list of buckets and also our test bucket `ce-rap-test`:
+
+![ce-rap-test](https://user-images.githubusercontent.com/1050582/62883470-c3ab9480-bd3c-11e9-9c7a-c77b171a0185.jpg)
+
+#### How to use rclone
+
+Some examples of rclone usage.
+
+List the contents of bucket `ce-rap-test`:
+
+```
+rclone ls openstax:ce-rap-test
+```
+
+Copy all content from `ce-rap-test` to your current directory:
+
+```
+rclone copy openstax:ce-rap-test ./
+```
+
+Copy one file into the bucket:
+
+```
+rclone copy ./nothinginside.md openstax:ce-rap-test/
+```
+
+Information: Be careful with `move` and `delete` commands!
+
+#### How to mount a bucket into your local filesystem with FUSE
+
+rclone can also mount buckets into your local filesystem with FUSE.
+
+Info: On mac you need to install [FUSE for macOS][macosfuse] first.
+
+First create a local folder e.g. in your home dir where you want to mount the bucket. For example:
+
+```
+mkdir ~/s3files
+```
+
+Now mount the bucket with rclone in the foreground:
+
+```
+rclone mount openstax:ce-rap-test ~/s3files
+```
+
+**Now you can use that folder as remote mount and can copy/move/delete files into s3 and out!**
+
+To stop mounting press `Ctrl-C`. Sometimes unmounting can fail. In this case use this command to unmount your mount folder manually:
+
+```bash
+# Linux
+fusermount -u /s3files
+# OS X
+umount /s3files
+```
+
+If you want o mount the bucket in the background you just need to add a `&` in the command:
```

Comment:
"want o" -> "want to"

Could you move this to before you talk about unmounting?  Together with your other `rclone mount` command line?

Reply:
done.